src/screens/SignUp/SignUp.js

```diff
@@ -1,14 +1,75 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import "./SignUp.css";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { userSignUp } from "../../services";
 
 const SignUp = () => {
+	const navigate = useNavigate();
+	const [username, setUsername] = useState("");
```

Comment:
suggestion: instead of having multiple useStates just have one with an object. something like this 
`const [loginData, setLoginData] = useState({ email: '', password: '' })`

Reply:
I'll update the states to a single state